Tests/SemanticVersioningTests/SemanticVersionTests.swift

```diff
@@ -24,37 +24,47 @@ class SemanticVersionTests: XCTestCase {
     }
     
     func testOrder() async throws {
-        //  1.0.0 < 2.0.0 < 2.1.0 < 2.1.1
-        let v100 = SemanticVersion(
+        //  1.2.0 < 2.0.0 < 2.0.1 < 2.1.0
+        let v120 = SemanticVersion(
             major: 1,
-            minor: 0,
+            minor: 2,
             patch: 0
         )
         let v200 = SemanticVersion(
             major: 2,
             minor: 0,
             patch: 0
         )
-        let v210 = SemanticVersion(
+        let v201 = SemanticVersion(
             major: 2,
-            minor: 1,
-            patch: 0
+            minor: 0,
+            patch: 1
         )
-        let v211 = SemanticVersion(
+        let v210 = SemanticVersion(
             major: 2,
             minor: 1,
-            patch: 1
+            patch: 0
         )
 
-        XCTAssertGreaterThan(v211, v210)
-        XCTAssertGreaterThan(v210, v200)
-        XCTAssertGreaterThan(v200, v100)
+        XCTAssertGreaterThan(v210, v201)
+        XCTAssertGreaterThan(v201, v200)
+        XCTAssertGreaterThan(v200, v120)
+        XCTAssertFalse(v210 < v201)
```

Comment:
You can use comparable-assetion with XCTExpectFailure.

```suggestion
        XCTExpectFailure {
          XCTAssertLessThan(v210, v201)
        }
```

Reply:
It seems better, thank you!
I'll use it.

Reply:
No need to that. 👍